views/building-hours/index.js

```diff
@@ -13,16 +13,29 @@ import type {BuildingInfoType} from './types'
 import hoursData from '../../data/building-hours.json'
 
 const buildingImages = {
-  pausekitchen: require('../../data/images/buildinghours/pausekitchen.small.jpeg'),
-  bookstore: require('../../data/images/buildinghours/bookstore.small.jpeg'),
-  convenience: require('../../data/images/buildinghours/convenience.small.jpeg'),
+  pause: require('../../data/images/buildinghours/pause.jpg'),
+  bookstore: require('../../data/images/buildinghours/bookstore.jpg'),
+  convenience: require('../../data/images/buildinghours/convenience.jpg'),
   postoffice: require('../../data/images/buildinghours/postoffice.small.jpeg'),
   rolvaag: require('../../data/images/buildinghours/rolvaag.small.jpeg'),
   halvorson: require('../../data/images/buildinghours/halvorson.small.jpeg'),
-  skoglund: require('../../data/images/buildinghours/skoglund.small.jpeg'),
-  cage: require('../../data/images/buildinghours/cage.small.jpeg'),
+  skoglund: require('../../data/images/buildinghours/skoglund.jpg'),
+  cage: require('../../data/images/buildinghours/cage.jpg'),
   stav: require('../../data/images/buildinghours/stav.small.jpeg'),
-  disco: require('../../data/images/buildinghours/disco.small.jpeg'),
```

Comment:
Two notes: One, I _had_ explicitly renamed all of these to `.jpeg`… 😉 but I don't really care, and I think I like `.jpg` better.

Two, the `.small` copy was a shrunken version of the big version, because no devices need to load a 1000-px-wide image for portrait mode, and it was making my Android simulators and phone laggy when I opened that view. I can change that after we merge; I'll run them through ImageOptim anyway, like I always do when a new image is included.